crates/p2p/src/swarm/handle.rs

```diff
@@ -34,55 +34,69 @@ pub struct ErrDroppedMsgs(u64);
 
 impl Display for ErrDroppedMsgs {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "P2PHandle dropped {} messages", self.0)
+        write!(f, "GossipHandle dropped {} messages", self.0)
     }
 }
 
-/// Handle to interact with P2P implementation spawned in another async
-/// task. To create a new one, use [`super::P2P::new_handle`].
+/// Handle to receive an event from p2p
 #[derive(Debug)]
-pub struct P2PHandle {
-    /// Event channel. [`super::P2P`] struct will send events via the channel and [`P2PHandle`] is
-    /// expected to get them via receiving side of the channel.
-    events: broadcast::Receiver<Event>,
+pub struct ReqRespHandle {
+    events: mpsc::Receiver<ReqRespEvent>,
+}
 
-    /// Command channel. [`P2PHandle`] will send commands via it and [`super::P2P`] struct is
-    /// expected to do logic corresponding to specific [`Command`].
-    commands: mpsc::Sender<Command>,
+impl ReqRespHandle {
+    pub(crate) const fn new(events: mpsc::Receiver<ReqRespEvent>) -> Self {
+        Self { events }
+    }
 
-    /// The [`libp2p::identity::Keypair`]. Used for creating our own [`PeerId`], and if cfg option
-    /// `message-signing` is enabled, to sign messages with it.
-    keypair: Keypair,
-}
+    /// Gets the next event from P2P from events channel.
```

Comment:
Some docstrings have `.` others don't. Please normalize/standardize. I prefer with `.` in the end of the sentences.